addons/calendar/tests/test_event_recurrence.py

```diff
@@ -74,6 +74,24 @@ def test_weekly_interval_2(self):
             (datetime(2019, 11, 5, 8, 0), datetime(2019, 11, 7, 18, 0)),
         ])
 
+    def test_weekly_interval_2_week_start_sunday(self):
```

Comment:
:+1:

Reply:
Forgot to set back `lang.week_start = '1' ` :wink: